tests/jballerina-unit-test/src/test/resources/test-src/expressions/ternary/ternary-expr-negative.bal

```diff
@@ -11,4 +11,12 @@ public function main(string... args){
 
 function test1 (int value) {
     string s1 = value > 40 ? true : false ? "morethan40" : "lessthan20";
-}
\ No newline at end of file
+}
+
+function test2() {
+    int[] a = [];
+    a.push(true? 1: "");
```

Comment:
```suggestion
    a.push(true ? 1 : "");
```

Please fix wherever applicable.

Reply:
Also with the upcoming unreachable changes, this may result in a compilation error. Shall we assign true to a variable and use it? Please fix wherever applicable too.